test/lib/ex_aws/cloudformation_test.exs

```diff
@@ -0,0 +1,55 @@
+defmodule ExAws.CloudformationTest do
```

Comment:
You need a test that actually hits the CloudFormation API, generally a basic list action.